src/coreclr/src/tools/Common/JitInterface/ThunkGenerator/Program.cs

```diff
@@ -480,15 +754,22 @@ static void Main(string[] args)
             else
             {
                 IEnumerable<FunctionDecl> functions = ParseInput(new StreamReader(args[0]));
-                using (TextWriter tw = new StreamWriter(args[1]))
-                {
-                    Console.WriteLine("Generating {0}", args[1]);
-                    WriteManagedThunkInterface(tw, functions);
-                }
-                using (TextWriter tw = new StreamWriter(args[2]))
+
```

Comment:
Seems like you should have at least minimal argument validation here, like checking the number of arguments, and a "usage" message that would also serve as a short description of what this tool is doing.